CMakeLists.txt

```diff
@@ -12,3 +12,8 @@ install(DIRECTORY launch
 install(PROGRAMS scripts/hydro-upgrade-notice
   DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}/
 )
+
+if(CATKIN_ENABLE_TESTING)
+  find_package(roslaunch REQUIRED)
+  roslaunch_add_file_check(test/camera_for_test.launch)
```

Comment:
@130s I also added `roslaunch_add_file_check(launch)` to check all `*.launch` files in `launch` directory.
Please note that `roslaunch_add_file_check` can check if all values of arguments are determined in a launch file, so checking launch files that contains `arg` without `default` attrinbute occurs failure on test (like: `<arg name="manager" />` at `launch/includes/depth.launch.xml`).
Also note that `roslaunch_add_file_check` with directory as argument only checks files with extension `*.launch`, so all `*.launch.xml` are currenly ignored.